templating/debug.rst

```diff
@@ -46,6 +46,8 @@ The same mechanism can be used in Twig templates thanks to ``dump()`` function:
         </a>
     {% endfor %}
 
-The variables will only be dumped if Twig's ``debug`` setting (in ``config.yml``)
-is ``true``. By default this means that the variables will be dumped in the
-``dev`` environment but not the ``prod`` environment.
+By design, the ``dump()`` function is only available if the ``debug`` setting
```

Comment:
I think we should make more clear what the `debug` setting is (one may confuse `twig.debug` and `kernel.debug` which are probably the same, but which still can differ).